pynbody/halo/__init__.py

```diff
@@ -25,6 +25,37 @@
 
 logger = logging.getLogger("pynbody.halo")
 
+class SimpleHalo(snapshot.ContainerWithPhysicalUnitsOption):
```

Comment:
I'm not sure I understand the difference between this and `DummyHalo`. Could it be explained somehow (is there perhaps a better name?)